proposals/0013-alignas.md

```diff
@@ -0,0 +1,283 @@
+<!-- {% raw %} -->
+
+# HLSL alignas Specifier
+
+* Proposal: [0013](0013-alignas.md)
```

Comment:
Use `NNNN` instead of `0013` until just before this merges, and you pick the next available number.